docs/imviz/plugins.rst

```diff
@@ -29,6 +29,16 @@ Plot Options
 
 This plugin gives access to per-viewer and per-layer plotting options.
 
+.. _imviz-subset-plugin:
+
+Subset Tools
+============
+
```

Comment:
Need to add a blurb here about "static regions" won't be listed.

https://github.com/spacetelescope/jdaviz/blob/398cb1d0dddfeb0c2716138678bd3ca376da2f94/jdaviz/configs/imviz/helper.py#L236